crates/oidc-client/src/requests/registration.rs

```diff
@@ -46,11 +59,23 @@ pub async fn register_client(
     http_service: &HttpService,
     registration_endpoint: &Url,
     client_metadata: VerifiedClientMetadata,
+    software_statement: Option<String>,
```

Comment:
Why is it not part of the VerifiedClientMetadata directly?

Reply:
That's because in my head, you could use `VerifiedClientMetadata` is also anything you could put as a claim in the software statement, so you can't put this field in it. But maybe that doesn't make sense :sweat_smile:.

Reply:
Yeah, makes sense. I'll probably touch that again once we actually start to care about the software_statement in MAS 👍